### PR TITLE
New package: Shssoichiro.Oxipng version 9.1.1

### LINKED ISSUE #151396: [Package Request]: oxipng
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/shssoichiro/oxipng
Publisher: shssoichiro
Package Name: oxipng
Description: Oxipng is a multithreaded lossless PNG/APNG compression optimizer.
Package Version: 9.1.1
Installer URL: https://github.com/shssoichiro/oxipng/releases/tag/v9.1.1

### PR DESCRIPTION
**Pull request by BiosPlus**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.2.1 :rocket:

- Resolves: #151396
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151434)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\oxipng.exe returned exit code: -1073741515
> Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\oxipng.exe returned exit code: -1073741515

(Automated response - build 877.)

**Comment by Trenly** [comment]
Has a dependency on VCRedist 2015+
image

File: manifests/s/Shssoichiro/Oxipng/9.1.1/Shssoichiro.Oxipng.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+PackageIdentifier: Shssoichiro.Oxipng
+PackageVersion: 9.1.1
+InstallerType: zip
+NestedInstallerType: portable
+UpgradeBehavior: install
+Commands:
+- oxipng
+ReleaseDate: 2024-04-23
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/s/Shssoichiro/Oxipng/9.1.1/Shssoichiro.Oxipng.installer.yaml:13]:
```suggestion
- Architecture: x86
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

File: manifests/s/Shssoichiro/Oxipng/9.1.1/Shssoichiro.Oxipng.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  NestedInstallerFiles:
+  - RelativeFilePath: oxipng-9.1.1-i686-pc-windows-msvc/oxipng.exe
+  InstallerUrl: https://github.com/shssoichiro/oxipng/releases/download/v9.1.1/oxipng-9.1.1-i686-pc-windows-msvc.zip
+  InstallerSha256: 9A181C0CB78A77492DABF102F1720FF263416B998BDA590FA77351B14FF4E411
+- Architecture: x64
```
Comment (Trenly) [inline manifests/s/Shssoichiro/Oxipng/9.1.1/Shssoichiro.Oxipng.installer.yaml:18]:
```suggestion
- Architecture: x64
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**